src/client_side_reply.cc

```diff
@@ -1484,17 +1485,18 @@ clientReplyContext::identifyStoreObject()
 {
     HttpRequest *r = http->request;
 
-    // client sent CC:no-cache or some other condition has been
-    // encountered which prevents delivering a public/cached object.
-    // XXX: The above text does not match the condition below. It might describe
-    // the opposite condition, but the condition itself should be adjusted
-    // (e.g., to honor flags.noCache in cache manager requests).
-    if (!r->flags.noCache || r->flags.internal) {
+    if (r->flags.noCache && !r->flags.internal) {
+        identifyFoundObject(nullptr, "no-cache"); // skip Store lookup for  "external" no-cache requests
+    } else if (ForSomeCacheManager(r->url.path())) { // cache manager requests ought to have flags.noCache
+        identifyFoundObject(nullptr, "no-cache"); // skip Store lookup for cache manager requests
```

Comment:
```suggestion
        identifyFoundObject(nullptr, "mgr"); // skip Store lookup for cache manager requests
```

Reply:
> we are using the second identifyFoundObject() parameter in Squid response headers (CACHE_STATUS?)

Yes, you are right.

>  and perhaps even some access.log codes?

I could not find such uses.

> PR description should reflect 

I added a sentence reflecting that.